airflow/models/taskinstance.py

```diff
@@ -1630,6 +1632,30 @@ def get(
                 """Get Airflow Variable after deserializing JSON value"""
                 return Variable.get(item, default_var=default_var, deserialize_json=True)
 
+        class ConnectionAccessor:
+            """
+            Wrapper around Connection. This way you can get connections in
+            templates by using ``{{ conn.conn_id }}`` or
+            ``{{ conn.get('conn_id', 'fallback') }}``.
+            """
+
+            def __getattr__(
+                self,
+                item: str,
+            ):
+                return Connection.get_connection_from_secrets(item)
+
+            @staticmethod
+            def get(
+                item: str,
+                default_conn: Any = None,
+            ):
+                """Get Airflow Connection value"""
+                try:
+                    return Connection.get_connection_from_secrets(item)
+                except AirflowNotFoundException:
+                    return default_conn
```

Comment:
Is it worth doing something like:


```suggestion
                    if isinstance(default_conn, str):
                        return Connection(uri=default_conn)
                    else:
                        return default_conn
```

to support passing a URI as the fallback and have it parsed to a Connection object?

Reply:
won't be that encouraging one of the security pitfalls ? Users then will write the URI in the dag source code and the URI most likely contains a secret, right?  Or did I misunderstood you

Reply:
Yeah, you're probably right.